src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
If you use frontend2 with the current version of arcade-machine in edge, arcade-machine will select the div that's enclosing the video cards, but not select the cards themselves